flycs_sdk/transformations.py

```diff
@@ -57,7 +59,9 @@ def __init__(
         :type version: str
         :param static: Whether or not the version should be appended to the table name, defaults to False
         :type static: bool, optional
-        :param has_output: Whether or not this query has a result that should be written into a table (false can be used to run DML or stored procedures for example), defaults to False
```

Comment:
@christophedcpm something weird happened here which reformatted the documentation